ledger-service/http-json/src/main/scala/com/digitalasset/http/Main.scala

```diff
@@ -82,7 +82,7 @@ object Main extends StrictLogging {
       case _ =>
     }
 
-    val serviceF: Future[HttpService.Error \/ ServerBinding] =
+    val serviceF: Future[HttpService.Error \/ (ServerBinding, Option[ContractDao])] =
```

Comment:
maybe worth having an `httpservice` class that contains the server binding & optional dao and abstracts away the details of closing? Not as part of the backport but separately